app/data/models/scheme.js

```diff
@@ -0,0 +1,23 @@
+module.exports = (sequelize, DataTypes) => {
```

Comment:
Consider renaming to something like `funding`.

Just because we have several `scheme` tables in the rest of the service that specifically relate to the scheme itself.  eg `schemeId: 1, name: SFI`

I also think we might need a `scheme` table here too.  Just so we can easily provide different logic for different schemes.  ie allow us to make a db to get all schemes including all their payment requests.  I'm not sure if I'm being too bias towards the setup of the existing repos with this comment though.

What do you think?

Reply:
Tots agree -- somehow just yeeted `schemeId` out from the `paymentRequests` coming into this service as not worthy of being saved down 😆